.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     'jest/globals': true
   },
   rules: {
+    'import/extensions': [WARNING, 'always', { ignorePackages: true }],
```

Comment:
i thought we could start with warnings on missing file extensions...but maybe we should go all the way to error? thoughts?

Reply:
Yeah, I like warnings.